dspace-api/src/main/java/org/dspace/scripts/configuration/ScriptConfiguration.java

```diff
@@ -105,6 +106,19 @@ public boolean isAllowedToExecute(Context context, List<DSpaceCommandLineParamet
      */
     public abstract Options getOptions();
 
+    /**
+     * The getter for the options of the Script (help informations)
+     * 
```

Comment:
Patch introduces trailing whitespace here. I wonder why checkstyle didn't catch that?

Reply:
I have no write access to the foreign backport branch. Please, can do a maintainer of dspace project this little change? A new/other PR for this is a little bit overhead...

Reply:
@ChrisBethgster oh, you are right! Yes, let me just fix that manually and merge this. Thanks!